packages/pages-shared/src/asset-server/metadata.ts

```diff
@@ -0,0 +1,66 @@
+// TODO: Use types from metadata-generator instead
+
+export type MetadataStaticRedirectEntry = {
+	status: number;
+	to: string;
+	lineNumber: number;
+};
+
+export type MetadataRedirectEntry = {
+	status: number;
+	to: string;
+	lineNumber?: number;
+};
+
+export type MetadataStaticRedirects = {
+	[path: string]: MetadataStaticRedirectEntry;
```

Comment:
At some point, we should union all of the index signature values with `undefined` so that dynamic property lookups are safer i.e. `[path: string]: MetadataStaticRedirectEntry | undefined`

Reply:
Yup. And this is probably the point where where we actually do bring together the metadata-generator and asset-server types.